website/projects/scimaps/src/app/shared/components/tile/tile.component.ts

```diff
@@ -0,0 +1,76 @@
+import { AfterViewInit, Component, NgZone, OnInit } from '@angular/core';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import * as OpenSeadragon from 'openseadragon';
+import { drawerInOut } from '../../../constants/drawer.animations';
+
+@Component({
+  selector: 'sci-tile',
+  templateUrl: './tile.component.html',
+  styleUrls: ['./tile.component.scss'],
+  animations: [drawerInOut]
+})
+export class TileComponent implements OnInit, AfterViewInit {
+
+  constructor(private zone: NgZone, private route: ActivatedRoute, private router: Router) { }
+  baseURL = 'https://scimaps.org/assets/map-tiles';
+  tile = 'map_of_the_internet_172';
+  show = false;
+  ngOnInit(): void {
+    this.show = true;
+    const parentSnapshot = this.route.parent?.snapshot;
+    const map = parentSnapshot?.data.map;
+    if (map) {
+      const segments = map.externalLink.split('/');
+      this.tile = segments[segments.length  - 2];
+    }
+  }
+
+  ngAfterViewInit(): void {
+    this.makeMap(
+      'map_canvas',
+      `${this.baseURL}/${this.tile}`, {
+        zoom_origin: 0,
+        min_viewable_zoom: 1,
+        max_viewable_zoom: 4,
+        background_color: '#333',
```

Comment:
Make the whole #map_canvas have background color: #333 via tile.component.scss as well